Assets/_Scripts/AIScripts/CatchPlayer.cs

```diff
@@ -25,7 +25,7 @@ public class CatchPlayer : MonoBehaviour
     private float _initEyeLightLevel;
 
     [Header("Misc")]
-    private bool _isCaught;
+    public bool _isCaught;
```

Comment:
So, this is mainly my coding convention but if you change a variable to public I usually change the name to upper camel case. This isn't obviously anything major.